.coveragerc

```diff
@@ -654,6 +657,7 @@ omit =
     homeassistant/components/wirelesstag/*
     homeassistant/components/xiaomi_aqara/*
     homeassistant/components/xiaomi_miio/*
+    homeassistant/components/xs1.py
```

Comment:
Please move the component to a package, ie its own directory under components. That's our latest way of structuring things.

Reply:
done